waku/waku_keystore/keystore.nim

```diff
@@ -183,13 +185,22 @@ proc getMembershipCredentials*(path: string,
     if jsonKeystore.hasKey("credentials"):
       # We get all credentials in keystore
       var keystoreCredentials = jsonKeystore["credentials"]
-      let key = query.hash()
-      if not keystoreCredentials.hasKey(key):
+      if keystoreCredentials.len == 0:
         # error
         return err(AppKeystoreError(kind: KeystoreCredentialNotFoundError,
-                                    msg: "Credential not found in keystore"))
+                                    msg: "No credentials found in keystore"))
+      var keystoreCredential: JsonNode
+      if keystoreCredentials.len == 1:
+        for v in keystoreCredentials.getFields().values():
+          keystoreCredential = v
```

Comment:
this is quite ugly - the compiler was throwing errors when i tried doing this -
```nim
keystoreCredential = keystoreCredentials.getFields().values()[0]
```

Reply:
Isn't that because values() returns iterator, which means you'd have to turn it into `seq` first before you can access the Nth element?

Reply:
thanks, addressed in 99b3fe6c!